docs/recipe_09.rst

```diff
@@ -0,0 +1,314 @@
+.. _recipe_09:
+
+======================================
+Reading and Writing ORCA Files
+======================================
+
+- ``import cctk`` is assumed.
+- *cctk* was originally designed with Gaussian in mind, but supports basic writing of ORCA input files and parsing of ORCA output
+- Note that the MiniPrint option may make Orca output files unreadable by cctk
+- All of the input and output files used in these recipes can be accessed at https://github.com/ekwan/cctk/tree/master/test/static
+- All of input files used in these recipes can also be copied as text at the end of this recipe.
+
+"""""""""""""""""""""""""""""""""""""""
+Writing a simple ORCA input file
+"""""""""""""""""""""""""""""""""""""""
+
+
+- In this recipe, we convert an ``.xyz`` file into an ORCA ``.inp`` file.
+- cctk can only write single-geometry input files for ORCA
+
+::
+
+    read_path = "test/static/test_peptide.xyz"
+    write_path = "test/static/test_peptide.inp"
+
+    file = cctk.XYZFile.read_file(read_path)
+    header = "! aug-cc-pVTZ aug-cc-pVTZ/C DLPNO-CCSD(T) TightSCF TightPNO MiniPrint\n%pal nproc 4 end\n%maxcore 4000\n%mdci\n    density none\nend"
+    cctk.OrcaFile.write_molecule_to_file(write_path, file.molecule, header)
+
+This returns the following ``.inp`` file.
+
+::
+
+  ! aug-cc-pVTZ aug-cc-pVTZ/C DLPNO-CCSD(T) TightSCF TightPNO
+  %maxcore 4000
+  %pal
+    nproc 4
+  end
+  %mdci
+    density none
+  end
+
+  * xyz 0 1
+  7         -2.59192634   -2.21198726    0.00000000
+  1         -1.59236634   -2.24165320    0.00000000
```

Comment:
I'd prefer not to have the recipes be too long, so I would just truncate most of this output with "..."